content/techniques/caching.md

```diff
@@ -70,27 +70,29 @@ You can also apply the `CacheInterceptor` to WebSocket subscribers as well as Mi
 ```typescript
 @@filename()
 @CacheKey('events')
+@CacheTTL(20)
```

Comment:
I don't think we should use `@CacheTTL` by default. Instead, a separate section should be created :)

Reply:
Fair enough. Will make.

Reply:
I've done what you suggested. 

In addition, I moved the "Customize caching" block to be above the "Websockets & Microservices" block, as I think it makes more sense in the context of the details inside of the caching details. Since the "Customize caching" bock specifies setting cache defaults (ttl & max), it makes more sense to read this before moving onto the CacheKey & CacheTTL blocks below it. If you don't think this makes sense, I can change it back, but to me this reads better.